### PR TITLE
Create ruckus-unleashed.yml

### LINKED ISSUE #417: [Missing SNMP Polling Profile]
[NOTE]: # ( ^^ Please add the Device Vendor and Model in the title above. ^^ )

Device Vendor: 
Commscope / Ruckus

Device Model: 
Ruckus Wireless H510

[NOTE]: # (If you do not know your SysOID, it can be gathered from the SNMP walk you provide at the end of this issue. It is the value of this OID in the SYSTEM-MIB: `1.3.6.1.2.1.1.2.0`)

System Object Identifier (SysOID): 
.1.3.6.1.4.1.25053.3.1.5.15

List of critical metrics that you expect in the profile (does not need to be all-inclusive, just a general idea of your expectations):

 - CPU
 - AP list and status
 - Mesh Network status
 - Client
 - network Traffic
 - wireless relates metrics - retransmist, signal strength
 
If there is a New Relic technical support case related to this request, please provide the case number:

**Please provide a [public gist](https://gist.github.com/) with a sanitized** *(sensitive information removed)* **SNMP walk of the device. Providing MIB files is not a substitute for an SNMP walk and failure to provide one here will result in delays for closing this issue. The output of these commands shows the "true story" of what a device will respond to in SNMP and is used to ensure that our profiles are efficient and accurate for your devices.**

Gist Link: 
https://gist.github.com/gitobic/d492ae9684c89fae22ddc0bddfc914d0

Also atta
---
Example SNMP walk commands:
```
# v2c example
snmpwalk -v 2c -t 10 -On -c $COMMUNITY_STRING $IP_ADDRESS . >> snmp_walk.out

# v3 example
snmpwalk -v3 -t 10 -l authPriv -u $USERNAME -a MD5|SHA -A $AUTH_PASS -x DES|AES -X $PRIV_PASS -ObentU -Cc $IP_ADDRESS . >> snmp_walk.out
```
[Ruckus-snmp-v2_walk.txt](https://github.com/kentik/snmp-profiles/files/9493217/Ruckus-snmp-v2_walk.txt)
[Unleashed_200-8-1--3-243_MIBs_20181105(1).zip](https://github.com/kentik/snmp-profiles/files/9493219/Unleashed_200-8-1--3-243_MIBs_20181105.1.zip)

### PR DESCRIPTION
**Pull request by Mesverrum**
The walk provided didn't appear to be complete but I did what I could with that plus the MIB

Resolves #417

### CODE REVIEW

File: profiles/kentik_snmp/ruckus/ruckus-unleashed.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.9.0
+      # Total transmitted fail packets of wireless interfaces
+      - name: ruckusUnleashedSystemStatsWLANTotalTxFail
+        OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.11.0
+      # Total transmitted retry packets of wireless interfaces
+      - name: ruckusUnleashedSystemStatsWLANTotalTxRetry
+        OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.12.0
+      # Number of registered AP
+      - name: ruckusUnleashedSystemStatsNumRegisteredAP
+        OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.15.0
```
Comment (thezackm) [inline profiles/kentik_snmp/ruckus/ruckus-unleashed.yml:56]: is this a duplicate of lines 30-32?

File: profiles/kentik_snmp/ruckus/ruckus-unleashed.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.16.0
+      # Number of All client devices
+      - name: ruckusUnleashedSystemStatsAllNumSta
+        OID: 1.3.6.1.4.1.25053.1.15.1.1.1.15.30.0
+
+metric_tags:
+  # Model
+  - name: 1.3.6.1.4.1.25053.1.15.1.1.1.1.9.0
+    OID: ruckusUnleashedSystemModel
+    tag: serial_number
```
Comment (thezackm) [inline profiles/kentik_snmp/ruckus/ruckus-unleashed.yml:68]: this tag is on the wrong metric_tag value

**Review by thezackm — CHANGES_REQUESTED** [review]

File: profiles/kentik_snmp/ruckus/ruckus-unleashed.yml (hunk 1 above)
Comment (Mesverrum) [reply profiles/kentik_snmp/ruckus/ruckus-unleashed.yml]: yep

File: profiles/kentik_snmp/ruckus/ruckus-unleashed.yml (hunk 2 above)
Comment (Mesverrum) [reply profiles/kentik_snmp/ruckus/ruckus-unleashed.yml]: fixed

**Review by thezackm — APPROVED** [review]

**Pull request merged**